### PR TITLE
fix:#549

### PR DESCRIPTION
**Pull request by RajuGangitla**
# ✨ Codu Pull Request 💻

## Pull Request details:
- Changed the "Heading 1" to "Heading" and "Heading 2" to "Subheading" and changed icons to Heading from Lucide 

## Associated Screenshots:

    
🚨 WIP - Things will break - Google Chrome 19-10-2023 23_14_00

### CODE REVIEW

File: package-lock.json
Comment (NiallJoeMaher) [inline package-lock.json:1]: Could you remove this file please? These changes shouldn't have changed the lock file. Everything else is great so once it's removed I'll get it merged. 🦾

**Review by NiallJoeMaher — CHANGES_REQUESTED** [review]

File: package-lock.json
Comment (NiallJoeMaher) [reply package-lock.json:1]: I think I mistook my words. I mean unstage the file (as in there should have been no changes to that file).

Comment (NiallJoeMaher) [reply package-lock.json:1]: I think you unstaged the wrong files 🙈 If you can revert I'll get this merged.

**Comment by NiallJoeMaher** [comment]
Let me know if you need help! 🤔 

Screenshot 2023-10-20 at 07 48 55

**Comment by RajuGangitla** [comment]
i merge the latest code into my forked code and now changed the code only in bubble menu 
can you please check now

File: components/editor/editor/components/node-selector.tsx [hunk 1]
```diff
@@ -39,23 +40,17 @@ export const NodeSelector: FC<NodeSelectorProps> = ({
...
-      icon: Heading1,
+      name: "Heading",
+      icon: Heading,
       command: () => editor.chain().focus().toggleHeading({ level: 1 }).run(),
       isActive: () => editor.isActive("heading", { level: 1 }),
     },
     {
-      name: "Heading 2",
-      icon: Heading2,
+      name: "SubHeading",
```
Comment (NiallJoeMaher) [inline components/editor/editor/components/node-selector.tsx]: Small typo and then I'll get it merged. Should be`Subheading` not `SubHeading`.

**Review by NiallJoeMaher — CHANGES_REQUESTED** [review]

**Review by NiallJoeMaher — APPROVED** [review]
LGTM! 🌮 Well done. 🚀

**Comment by RajuGangitla** [comment]
Thank you @NiallJoeMaher I made my first open source contribution. I'm looking forward to contribute more

**Pull request merged**